roles/neutron-data-network/tasks/main.yml

```diff
@@ -25,6 +25,18 @@
     name: iputils
   when: os == 'rhel'
 
+- name: install custom ucarp scripts - rhel
+  template:
+    src: "usr/local/libexec/{{ item }}"
```

Comment:
ah, here says it's different (nice):
https://www.rpmfind.net/linux/RPM/fedora/24/x86_64/u/ucarp-1.5.2-15.fc24.x86_64.html

Do we need to create /usr/local/libexec dir ?